src/test/java/uk/gov/hmcts/reform/sandl/snlrules/rules/DateTimeHelper.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.sandl.snlrules.rules;
+
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+
+public final class DateTimeHelper {
+    private DateTimeHelper() {
+    }
+
+    public static OffsetDateTime offsetDateTimeOf(String date) {
```

Comment:
why here as part of a separate class as it's the same as the one in DateTimeUtils.